src/routes/jira/atlassian-connect/jira-atlassian-connect-get.ts

```diff
@@ -66,6 +66,14 @@ const modules = {
 		conditions: adminCondition
 	},
 	generalPages: [
+		{
+			key: "create-branch-options",
+			name: {
+				value: "GitHub Create Branch"
+			},
+			url: `/create-branch-options?issueKey={ac.issueKey}&issueSummary={ac.issueSummary}`,
+			location: "none"
+		},
```

Comment:
What is the use of general pages?

Reply:
This is so we are a connect app page inside the iframe. no iframe no jirahost.